content/en/user-guide/aws/servicediscovery/index.md

```diff
@@ -175,3 +175,30 @@ $ awslocal servicediscovery list-instances \
 The output will consist of the resource ID, and you can further use the [`DiscoverInstances`](https://docs.aws.amazon.com/cloud-map/latest/api/API_DiscoverInstances.html) API. This API allows you to query the DNS records associated with the service and perform various operations.
 
 To explore the DNS records of your service and perform other operations, refer to the [AWS CLI documentation](https://docs.aws.amazon.com/cli/latest/reference/servicediscovery/index.html) for comprehensive instructions and examples.
+
+### Using filters
+Filters can be used to narrow down the results of a list operation. Filters are supported for the following operations:
+
+- list-namespaces
+- list-services
+- discover-instances
+
+Using `list-namespaces` you can filter for the parameters `TYPE`, `NAME`, `HTTP_NAME`. Using `list-services` it is only possible to filter for `NAMESPACE_ID`.
+Both `list-services` and `list-namespaces` support `EQ` (default condition if not specified) and `BEGINS_WITH` as conditions. Both conditions and only support a single value to match by. The following examples demonstrate how to use filters with these operations:
```

Comment:
```suggestion
Using `list-namespaces` you can filter for the parameters `TYPE`, `NAME`, `HTTP_NAME`.
Using `list-services` it is only possible to filter for `NAMESPACE_ID`.
Both `list-services` and `list-namespaces` support `EQ` (default condition if not specified) and `BEGINS_WITH` as conditions.
Both conditions and only support a single value to match by.
The following examples demonstrate how to use filters with these operations:
```

I believe that the standard is to keep one sentence in one line for easier readability. I was unable to find where in notion where it's mentioned. :(

Reply:
fixed for complete do in: https://github.com/localstack/docs/pull/1142/commits/2966f96624d439a148cd69c97c528511b9ce262b